stable_baselines3/common/evaluation.py

```diff
@@ -52,14 +52,13 @@ def evaluate_policy(
     """
     is_monitor_wrapped = False
     # Avoid circular import
-    from stable_baselines3.common.env_util import is_wrapped
     from stable_baselines3.common.monitor import Monitor
 
-    if isinstance(env, VecEnv):
-        assert env.num_envs == 1, "You must pass only one environment when using this function"
-        is_monitor_wrapped = is_vecenv_wrapped(env, VecMonitor) or env.env_is_wrapped(Monitor)[0]
-    else:
-        is_monitor_wrapped = is_wrapped(env, Monitor)
+    if not isinstance(env, VecEnv):
+        gym_env = env
```

Comment:
why not `env = DummyVecEnv([lambda: env])` directly?

also please add a comment of why.

Reply:
I guess it works to do it that way. I was just unnecessarily nervous because using lambdas without care has screwed me over before. Changed.